source/components/composing-components.md

```diff
@@ -1,8 +1,11 @@
-Components really shine when you use them to their full potential, that is, by composing them with one another.
-Take for example the `<ul>` element, and the fact that only `<li>` elements are appropriate as children.
-If we want the same type of behavior, then we have to compose our components.
+Components organize applications best when leveraged with composition.
```

Comment:
I was confused by the sentence at first read. Maybe flip it, "applications
are best organised when"
On 26 Aug 2015 13:01, "Matthew Beale" <notifications@github.com> wrote:

> In source/components/composing-components.md
> <https://github.com/emberjs/guides/pull/666#discussion_r37973453>:
>
> > @@ -1,8 +1,11 @@
> > -Components really shine when you use them to their full potential, that is, by composing them with one another.
> > -Take for example the `<ul>` element, and the fact that only `<li>` elements are appropriate as children.
> > -If we want the same type of behavior, then we have to compose our components.
> > +Components organize applications best when leveraged with composition.
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/emberjs/guides/pull/666/files#r37973453>.
>